tests/pds/pds_postman_test.go

```diff
@@ -13,16 +13,17 @@ var _ = Describe("{RunPdsPostManApiLoadTests}", func() {
 	})
 	It("Deploy Dataservices", func() {
 		Step("Starting to execute Postman-Newman on PDS", func() {
-			var resultsFileName = "postman_results"
+			var resultsFileName = "result_" + RandomString(5) + ".json"
```

Comment:
Can we use timestamp here instead of `RandomString`? That way you can get the info about when the file was generated.